src/applications/bmqstoragetool/m_bmqstoragetool_printer.cpp

```diff
@@ -75,23 +75,23 @@ void printDataFileMeta(bsl::ostream&                 ostream,
                        const mqbs::DataFileIterator* dataFile_p,
                        bslma::Allocator*             allocator)
 {
-    BSLS_ASSERT_SAFE(dataFile_p && dataFile_p->isValid());
+    if (!dataFile_p || !dataFile_p->isValid()) {
+        return;  // RETURN
+    }
 
-    const bsl::vector<const char*> fields = {"BlazingMQ File Header",
+    const bsl::vector<bsl::string> fields = {"BlazingMQ File Header",
                                              "Data File Header"};
 
-    PRINTER_TYPE1 printer(ostream, &fields);
+    PRINTER_TYPE1 printer(ostream, fields);
     {
         bmqu::MemOutStream s(allocator);
-        s << '\n';
         mqbs::FileStoreProtocolPrinter::printFileHeader<PRINTER_TYPE2>(
             s,
             *dataFile_p->mappedFileDescriptor());
         printer << s.str();
     }
     {
         bmqu::MemOutStream s(allocator);
-        s << '\n';
```

Comment:
It seems it is discarded change from PR https://github.com/bloomberg/blazingmq/pull/665.